libsrc/pce/psg.s

```diff
@@ -0,0 +1,29 @@
+
+        .include        "pce.inc"
+
+        .export         psg_init
+
+psg_init:
+        clx
+        stx     $0800           ; Select channel
+psg_clear_loop:
+        stz     $0801           ; Clear global balance
```

Comment:
That line belongs outside the loop (it controls the sound _after all six_ channels have been mixed together).